datalad/cmd.py

```diff
@@ -335,10 +374,19 @@ def __call__(self,
                         self.runner = None
 
         except CommandError as command_error:
-            # The command exited with a non-zero return code
-            lgr.error("%s: command error: %s", self, command_error)
-            self.return_code = command_error.code
+            # Convert CommandError into BatchedCommandError
             self.runner = None
+            self.return_code = command_error.code
+            raise BatchedCommandError(
```

Comment:
I think this should be a `raise BatchedCommandError(...) from command_error` to not loose the original context/traceback.